core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -120,16 +120,17 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
                               spendProfile: Option[ProfileModel],
                               billingProjectNamespace: String,
                               ctx: RawlsRequestContext,
-                              location: Option[String]
```

Comment:
Location isn't used anywhere (I think it was before location was tied to the billing project), so I went ahead and removed it.

Reply:
Cloning an Azure workspace through `MultiCloudWorkspaceSerivce` now always passes a policies argument. But I did not add policies to the clone call for a Rawls stage workspace in `WorkspaceService`. Can anyone think of a reason why we should pass it there too?

Reply:
The only use case I can think of is a GCP workspace with a snapshot getting cloned and the user clicking the PHI button -- but I don't think that's UI we support atm. So, no, not right now.